api/api.go

```diff
@@ -270,6 +270,10 @@ func (c *Client) Status(trackID string) (*StatusInfo, error) {
 
 	defer res.Body.Close()
 
+	if _, err := c.List(trackID); err != nil {
+		return nil, err
+	}
+
```

Comment:
Sorry, I should have been more specific.

This is the endpoint I was referencing:

`http://exercism.io/api/v1/tracks/elixir/status?key=<someKey>`